src/js/utils/html-utils.js

```diff
@@ -76,19 +113,39 @@
          *
          * @param {string} formatString The string to be interpolated.
          * @param {Object} parameters An optional set of parameters to the template.
```

Comment:
The word "template" seems odd here.  Instead, maybe "...parameters for interpolation."